spec/unit/actions/associate_clients_spec.rb

```diff
@@ -92,4 +92,32 @@
       expect { GoodData::LCM2.run_action(GoodData::LCM2::AssociateClients, params) }.to raise_error(/The parameter/)
     end
   end
+
+  context 'in any delete mode' do
+    let(:params) do
```

Comment:
Identical code found in associate_clients_spec.rb:36, associate_clients_spec.rb:97 (mass = 76)